logging.go

```diff
@@ -89,6 +104,21 @@ func init() {
 
 	// Setting default LogFile
 	SetLogFile(defaultLogFile)
+
+	// Create the default prefixer
+	defaultPrefix := &defaultPrefixer{
+		prefixFormat: "%s [%s] ",
+		timeFormat:   defaultTimestampFormat,
+	}
+	SetPrefixer(defaultPrefix)
+}
+
+func (p *defaultPrefixer) CreatePrefix(loggingLevel Level) string {
+	return fmt.Sprintf(p.prefixFormat, time.Now().Format(p.timeFormat), loggingLevel)
+}
+
+func SetPrefixer(p Prefixer) {
```

Comment:
should we also have a "SetDefaultPrefixer()" function to set the default prefix in case we want to get back to default ?